### DISCUSSION TITLE
Template variable in dependency dashboard footer

### DISCUSSION THREAD

**Discussion by ademar59** [renovatebot/renovate#13319] [Request Help] [answered]
Hi,

I try to add a dependencyDashboardFooter with template variable and it doesnt work.

``` 
 "dependencyDashboardFooter": "You can run on demand scan with this [pipeline](https://gitlab.com/mobivia/driverservices/cloud-platform/digipaas-components/renovate/renovate/-/pipelines/new?var[RENOVATE_ONBOARDING]=false&var[RENOVATE_EXTRA_FLAGS]={{project}})",
``` 

Is this feature available ?1

**Comment by rarkins** [comment]
https://docs.renovatebot.com/templates/

**Reply by ademar59** [reply]
prBodyTemplate with is structur is easily customizable : {{header}}}{{{table}}}{{{notes}}}{{{changelogs}}}{{{configDescription}}}{{{controls}}}{{{footer}}}

It is not the same for "onboardingPr" and "dependencyDashboard".

My needs is to add message for onboarding and dependencyDasboard with project path variable.
This message add a link to run a new pipeline (onboarding=true or false depending the case).

Scan is scheduled every 4 hours (for all our projects) and with this link, people can run an extra scan limited to one project.

Do you think it will be possible to have those features ? 

Unfortunately, Gitlab markdown have no feature to add varaibles in .md : https://gitlab.com/gitlab-org/gitlab/-/issues/18103

**Reply by rarkins** [reply]
These features of customising onboarding PR or dependency dashboard are not currently supported. You are welcome to raise feature requests for each separately if no existing ones exist

**Comment by ademar59** [comment, marked as answer]
I have created 2 issues

**Comment by ademar59** [comment]
i have submit a pull request for that https://github.com/renovatebot/renovate/pull/13598
